test/controller/objecting.entity.name.controller.spec.unit.ts

```diff
@@ -478,6 +483,86 @@ describe("objecting entity name tests", () => {
   });
 });
 
+it("should receive error messages when no company name is entered with feature flag true", async () => {
```

Comment:
Can this get changed to `should receive error messages when no name or company name is entered when the feature flag is set to true`